test/library-tests.ts

```diff
@@ -128,7 +128,12 @@ describe('Library directories (c++)', () => {
             undefined,
             '/tmp/compiler-explorer-compiler-123',
         );
-        expect(qtpaths).toContain('-L/tmp/compiler-explorer-compiler-123/qt/lib');
+
+        if (process.platform === 'win32') {
+            expect(qtpaths).toContain('-L\\tmp\\compiler-explorer-compiler-123\\qt\\lib');
```

Comment:
(ditto all these could use the normalisePath thing if we made a test-only thing:
```expect(normalisePath(qtpaths)).toContain(/unix/path/)```
or maybe it'd need to handle lists, or we could do a fairly straightforward mapping thing